docs/src/pages/components/tables/SpanningTable.js

```diff
@@ -54,11 +54,17 @@ export default function SpanningTable() {
     <Paper className={classes.root}>
       <Table className={classes.table} aria-label="spanning table">
         <TableHead>
+          <TableRow>
+            <TableCell align="center" colSpan={3}>
+              Detail
```

Comment:
vs Details 🤔 ?

Reply:
haha details if you're a perfectionist

Reply:
Alright, let's go for it, do you want to update the pr? :)